### PR TITLE
RCLL-202 Recall recorded

### PR DESCRIPTION
**Pull request by danbenton-mojdt**

### CODE REVIEW

File: server/views/pages/recall/recall-recorded.njk [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <div class='govuk-grid-column-two-thirds'>
+
+          {{ govukPanel({
+            titleText: "Recall recorded"
+          }) }}
+
+          <div class="govuk-!-margin-top-6">
+            <h2 class="govuk-heading-l">What you can do next</h2>
+
+            <p><a href="{{ urls.adjustments }}">Check the adjustments information</a></p>
```
Comment (carloveo-moj) [inline server/views/pages/recall/recall-recorded.njk:18]: does a recall user always have the adjustments role? I assume you'd only want to show the adjustment link to users who can use it?

**Review by carloveo-moj — DISMISSED** [review]
LGTM

**Review by carloveo-moj — APPROVED** [review]
LGTM

File: server/views/pages/recall/recall-recorded.njk (hunk 1 above)
Comment (danbenton-mojdt) [reply server/views/pages/recall/recall-recorded.njk:18]: Have picked up with BAs - the application object in config is an object to allow for the required roles to be listed if that's the case

**Pull request merged**